ui/app/templates/vault/cluster/requests.hbs

```diff
@@ -6,9 +6,6 @@
   </p.levelLeft>
 </PageHeader>
 
-{{!--
-  <HttpRequestsBarChart @counters=counters />
---}}
-
+<HttpRequestsBarChart @counters={{model.counters}}/>
```

Comment:
The chart should be hidden if there's only one month of data (since there's nothing to compare yet)